.github/workflows/benchmark1.yaml

```diff
@@ -67,7 +67,7 @@ jobs:
 
       - name: Populate python deps
         run: |
-          echo -e "pandas\ngrip>=4.6.0\nmatplotlib>=3.7.0\nnumpy>=1.22.0\nseaborn>=0.12.0\nkubernetes>=28.0.0" > requirements.txt
+          echo -e "pandas\ngrip>=4.6.0\nmatplotlib>=3.7.0\nnumpy>=1.22.0\nseaborn>=0.12.0\nkubernetes>=28.0.0\npykube\nkubernetes-asyncio\nGitPython" > requirements.txt
```

Comment:
Is this supposed to include all the package included in `python_deps` in `setup/install_deps.sh` (~ line 138)?  I see also `PyYAML`.  If so, we are duplicating the list in two places. Perhaps time to make `requirements.txt` directly and have both read it?